src/styles/theme.scss

```diff
@@ -0,0 +1,124 @@
+// Color Palette
+$white: #fff;
+$black: #000;
+$cyan: #25CED1;
+$orange: #FF8360;
+$green: #2AC27F;
+$red: #E52C2C;
+$yellow: #FFBE0B;
+$gray: #737581;
+
+$gray-darker:  #434551;
+$gray-dark: #60626E;
+$gray-medium: #888A94;
+$gray-light: #9D9EA7;
+$gray-lighter: #B9BAC0;
+$gray-very-light: #D5D6D9;
+$gray-lightest: #EEEEF0;
+
+// Default Variables
+$primary-color: $cyan;
+$secondary-color: $orange;
+$site-content-text-color: $gray;
+$standard-label-color: $gray-light;
+$placeholder-color: $gray-lighter;
+$input-value-color: $gray-medium;
+$help-text-color: $gray-light;
+$disables-opacity: 0.25;
+$success-color: $green;
+$info-color: $orange;
+$warning-color: $yellow;
+$danger-color: $red ;
+$main-font-family: 'Lato', sans-serif;
+$primary-background-color: #F5F8FA;
+$light-color: $gray-very-light;
+$dark-color: $gray;
+
+
+/* Title - Lato B48 */
+
+.title {
```

Comment:
@michaelbukachi With how you have set this up does this mean we can call `.title` in our <style> section of our components to use the features under style? If not, is there a way to be able to define something like `main-font-family-title` as a variable and call it across the app?

Reply:
To apply styling for the `title`, setting the class of a tag to `title` should do it. e.g `<span class="title">Some tile</span>`